src/components/InterviewScheduleModal.tsx

```
@@ -0,0 +1,411 @@
+                <option value='morning'>Morning (9 AM - 12 PM)</option>
+                <option value='afternoon'>Afternoon (12 PM - 5 PM)</option>
+                <option value='evening'>Evening (5 PM - 8 PM)</option>
+                <option value='flexible'>Flexible</option>
+              </select>
+            </div>
+
+            {/* Preferred Days */}
+            <div>
+              <label className='block text-sm font-medium text-dark mb-2'>
+                Preferred Days *
+              </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)